pkg/controller/iam/iamclient/iamclient_integration_test.go

```diff
@@ -53,14 +53,19 @@ func init() {
 	// run-tests and skip-tests allows you to limit the tests that are run by
 	// specifying regexes to be used to match test names. See the
 	// formatTestName() function to see what test names look like.
-	flag.StringVar(&runTestsRegex, "run-tests", "", "run only the tests whose names match the given regex")
+	flag.StringVar(&runTestsRegex, "run-tests", "", "run the tests whose names match the given regex if the tests are allowed")
 	flag.StringVar(&skipTestsRegex, "skip-tests", "", "skip the tests whose names match the given regex, even those that match the run-tests regex")
+
+	// must-run-tests triggers the tests that match the given regex regardless
+	// of other "shouldRun" conditions.
+	flag.StringVar(&mustRunTestsRegex, "must-run-tests", "", "run and only run the tests whose names match the given regex if the regex is specified")
```

Comment:
What problem are we solving?  Can we use the unified tests instead?

Reply:
Without the new flag, we can't trigger a specific test case if it is not within the scope of the "shouldRun". Adding this flag to add more flexibility for local testing.